BEIMA.Client/src/pages/Users/AddUserPage.js

```diff
@@ -0,0 +1,86 @@
+import styles from './AddUserPage.module.css';
+import { useOutletContext, Link } from 'react-router-dom';
+import { IoArrowBack } from "react-icons/io5";
+import { Card, Button, Form, Row, Col } from 'react-bootstrap';
+import FormListWithErrorFeedback from '../../shared/FormList/FormListWithErrorFeedback.js';
+import { useEffect, useState } from "react";
+
+const AddUserPage = () => {
+  const mandatoryUserFields = {
```

Comment:
I was going to suggest changing this to allUserFields, but maybe a suggested improvement could be that certain roles might have unique fields, in which this variable name is fine